mne/evoked.py

```diff
@@ -914,6 +914,7 @@ def get_peak(
         time_as_index=False,
         merge_grads=False,
         return_amplitude=False,
+        strict=True,
```

Comment:
We try to add these when not present when adding new kwargs these days
```suggestion
        *,
        strict=True,
```